Lib/test/test_sqlite3/test_factory.py

```diff
@@ -66,7 +66,15 @@ class Factory(sqlite.Connection):
             def __init__(self, *args, **kwargs):
                 super(Factory, self).__init__(*args, **kwargs)
 
-        con = sqlite.connect(":memory:", 5.0, 0, None, True, Factory)
+        regex = (
+            r"Passing more than 1 positional argument to _sqlite3.Connection\(\) "
```

Comment:
If we want the deprecation warning to be slightly nice for the user, we can apply gh-107879, so the deprecation message will use `sqlite3.Connection` instead of `_sqlite3.Connection`.

Reply:
After de7e87cc0733b90c2693becac059dd72b58e925a, I don't think this is needed.